master/stream.py

```diff
@@ -1,42 +1,52 @@
+import subprocess
+
 import lomond
 from lomond import events
 
-url = "http://localhost:8080"
+from determined.experimental import client
+
+url = "localhost:8080"
+
 
 # steal determined token
-from determined.experimental import client
+subprocess.call(["det", "user", "login"])
```

Comment:
instead of using subprocess.call, delete the `url` variable I think.

Reply:
instead of what?

Reply:
url is used in both: 
```
client.login(url)
```
and
```
lomond.WebSocket(f"{url}/stream")
```

Are we wanting to replace those?